result/block-scoped-var.js

```diff
@@ -0,0 +1,57 @@
+/*eslint block-scoped-var: "error"*/
+
+function doIf() {
+    if (true) {
+        var build = true;
+    }
+
+    console.log(build);
```

Comment:
__[development][eslint]__
* Error - Unexpected console statement. (no-console)
* Error - 'console' is not defined. (no-undef)
* Error - 'build' used outside of binding context. (block-scoped-var)